app/scripts/app.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-angular
+var app = angular
   .module('openhimWebui2App', [
```

Comment:
Maybe we should take this opurtunity to rename the module, its was pretty badly named (by me :) ) Maybe we can make this openhimConsoleApp?